libgoal/libgoal.go

```diff
@@ -435,6 +435,25 @@ func (c *Client) SendPaymentFromWallet(walletHandle, pw []byte, from, to string,
 		return transactions.Transaction{}, err
 	}
 
+	if err != nil {
```

Comment:
Why the duplicate `if` statement and return?

Reply:
copy paste/self-merge error